### DISCUSSION TITLE
Postgres supported version? Help with manual migration from older version

### DISCUSSION THREAD

**Discussion by Rathna-K** [teslamate-org/teslamate#1990] [Q&A]
Hello, I tried pulling a new image of postgres (13.4) and it fails to upgrade the DB from its current version (12.2). any thoughts on how to tackle this problem? TIA -R

**Comment by instantdreams** [comment]
I suspect this might be related to my issue #1988

**Comment by Rathna-K** [comment]
Not sure. I ended up dumping the DB and reimporting it in to a new instance.
